src/material-experimental/mdc-slider/harness/slider-harness.ts

```diff
@@ -40,9 +40,16 @@ export class MatSliderHarness extends ComponentHarness {
     return id !== '' ? id : null;
   }
 
-  /** Gets the current display value of the slider. */
-  async getDisplayValue(): Promise<string> {
-    return (await this._textLabel()).text();
+  /**
+   * Gets the current display value of the slider. Returns null if the thumb
+   * label is disabled.
+   */
+  async getDisplayValue(): Promise<string|null> {
```

Comment:
@jelbourn For consistency with the MDC slider harness, I've updated the standard slider harness to also return "null" if the slider has the "thumbLabel" disabled. Currently as a side-effect of how the label is implemented, the `getDisplayValue` function also works if thumb labels are disabled. What do you prefer?

Reply:
I think that's fine